instana/recorder.py

```diff
@@ -63,7 +63,8 @@ def queued_spans(self):
             except queue.Empty:
                 break
             else:
-                spans.append(span)
+                if span.l != 0:
+                    spans.append(span)
```

Comment:
Would it make sense instead of filtering out suppressed spans on read not to store them at all? With current implementation the queue length returned by `queue_size()` does not match the actual number of elements returned by `queued_spans()`.

Reply:
Absolutely true. I should fix that.

Reply:
Changed.